terminfo/k/konsole/term.go

```diff
@@ -34,7 +34,7 @@ func init() {
 		EnterAcs:     "\x0e",
 		ExitAcs:      "\x0f",
 		EnableAcs:    "\x1b)0",
-		StrikeOut:    "\x1b[9m",
+		StrikeThrough:"\x1b[9m",
```

Comment:
It's a nit, but this (and the other impacted terminals) should probably have been reformatted with go fmt.  There should be a space between the ":" and the value.

Don't worry about it though, I'll regenerate them later anyway.